astropy/coordinates/tests/test_iau_fullstack.py

```diff
@@ -107,11 +107,11 @@ def test_iau_fullstack(fullstack_icrs,  fullstack_fiducial_altaz,
     assert np.all(addecs < tol), 'largest Dec change is {0} mas, > {1}'.format(np.max(addecs.arcsec*1000), tol)
 
     # check that we're consistent with the ERFA alt/az result
-    xp, yp = u.Quantity(iers.IERS.open().pm_xy(fullstack_times.jd1, fullstack_times.jd2)).to(u.radian).value
+    xp, yp = u.Quantity(iers.IERS.open().pm_xy(fullstack_times.utc.jd1, fullstack_times.utc.jd2)).to(u.radian).value
```

Comment:
I'm thinking that it's always *possible* that ``fullstack_times`` might get non-UTC times in the future, so better to be safe.  But you're right that I can just use the same approach as the other places (ditto below)